test/controllers/update/update.manage.trusts.review.individuals.controller.spec.ts

```diff
@@ -146,6 +146,70 @@ describe('Update - Manage Trusts - Review individuals', () => {
       expect(resp.text).toContain('Beneficiary');
     });
 
+    test('Page displays status as active when entity is still involved in the trust', async () => {
```

Comment:
I would change the name/word in the test description here from 'entity' to 'individual trustee'. Just makes things a bit clearer and avoids confusion with an 'overseas entity'. Same in the test below.

Reply:
Changed this to be more descriptive